src/frontend/views/mixins/filters-group.pug

```diff
@@ -0,0 +1,24 @@
+mixin filtersGroup(prop)
+  case prop.type()
+    when 'string'
+      .filter
+        label.label= `${prop.name()} contain:`
```

Comment:
h._.startCase(prop.name()) - and this goes everywhere where prop.name() is used as a label

Reply:
ok